### DISCUSSION TITLE
Why the window post starts very late in the deadline

### DISCUSSION THREAD

**Discussion by de-authority** [filecoin-project/lotus#6986] [Storage Provider] [answered]
Even though it still passed, it was dangerous (this quesion are select from the slack channel)

**Comment by de-authority** [comment, marked as answer]
actually nothing special happened, just busy sealing.
Reduce ParallelFetchLimit in miner config.toml will help. Default value 10

**Comment by dd45e640b42e6da7da96faee3996ef7c** [comment]
it needs to be noted that there can be plenty of reasons why a wdPost starts late. for example:

- problems connecting to the store storage
- an out of sync node
-....

**Reply by de-authority** [reply]
@f8-ptrk Thank you for adding.  but i think you will miss the window post if you have this two problems.
